.github/workflows/daily.yaml

```diff
@@ -0,0 +1,57 @@
+name: daily
+on:
+  # build every weekday at 4:00 AM UTC
+  schedule:
+    - cron: '0 4 * * 1-5'
```

Comment:
this is unnecessarily culture specific. run it every day. people legitimately work on the weekends/don't on weekdays.

Reply:
That's a perspective I haven't encountered before and entirely legitimate. I'll include the change in my follow up.